app/assets/stylesheets/partials/_footer.scss

```diff
@@ -7,10 +7,21 @@ footer {
   font-family: $font_family-secondary;
   color: $color_font-light;
   line-height: $font_size-small;
+  transition-duration: 1.5s;
 
+  // Override normal grey headings
+  h3 {
```

Comment:
The footer shouldn't be using `h3` tags. h3 tags should follow h2 tags which should follow h1 tags, etc. They don't make sense in the context of the footer. These can be spans.